messaging.rst

```diff
@@ -265,13 +265,14 @@ Mediated Transfer
 
 A :term:`Mediated Transfer` is a hash-time-locked transfer. Currently raiden supports only one type of lock. The lock has an amount that is being transferred, a :term:`secrethash` used to verify the secret that unlocks it, and a :term:`lock expiration` to determine its validity.
 
-Mediated transfers have an :term:`initiator` and a :term:`target` and a number of hops in between. The number of hops can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of hops a mediated transfer will require ``6N + 8`` messages to complete. These are:
+Mediated transfers have an :term:`initiator` and a :term:`target` and a number of mediators in between. The number of mediators can also be zero as these transfers can also be sent to a direct partner. Assuming ``N`` number of mediators, a mediated transfer will require ``10N + 14`` messages to complete. These are:
 
 - ``N + 1`` mediated or refund messages
 - ``1`` secret request
-- ``N + 1`` secret reveal
-- ``N + 1`` secret
-- ``3N + 4`` ACK
+- ``N + 2`` secret reveal
+- ``N + 1`` unlock
+- ``2N + 2`` processed (for mediated or refund messages, and unlock messages)
```

Comment:
Actually it's for all messages except `Delivered`, `Ping` and `Pong`.

Reply:
Then the number was also wrong.  Fixed.